### ISSUE TITLE
GoldHEN 2.2 - CheatMenu

### ISSUE THREAD

**Issue by DevSharpOfficial** [GoldHEN/GoldHEN#34]
Hi! First of all, Thank you for the new version of GoldHEN. I have an issue with the new cheatmenu, everytime i try to activate cheats on any games i have my PS4 is crashing and shutting down. After i try to start the console again its loading the storage memory status and then boot up. I hope you can help me out with this problem. Best Regards.

**Comment by alberhazky** [comment]
Having the same exact problem here, I have tried it in 3 different games, I activate the cheat and as soon as the game starts or tries to start,  the console shuts down, not sure if this is a kernel panic, I've tried going back to using the regular trainers and it works fine

**Comment by serpent24** [comment]
Bonsoir , Comment on installe le cheatMenu , 
Merci!

**Comment by Masamune3210** [comment]
Update to the latest version of GoldHEN

**Comment by SiSTR0** [comment]
Hi! send contact me on discord, I will check

**Comment by alberhazky** [comment]
ᛒᚱᚪᚾᛞᚩᚾ ᚪᛚᛒᛖᚱᚻᚪᛋᚳᛁ#1402

**Comment by alberhazky** [comment]
> Hi! send contact me on discord, I will check

ᛒᚱᚪᚾᛞᚩᚾ ᚪᛚᛒᛖᚱᚻᚪᛋᚳᛁ#1402

**Comment by wawan626385** [comment]
> Hai! Pertama-tama, Terima kasih atas versi baru GoldHEN. Saya memiliki masalah dengan menu cheat baru, setiap kali saya mencoba mengaktifkan cheat pada game apa pun, PS4 saya mogok dan mati. Setelah saya mencoba memulai konsol lagi, itu memuat status memori penyimpanan dan kemudian boot. Saya harap Anda dapat membantu saya dengan masalah ini. Salam Hormat.

coba di install versi 9.00 nya,

**Comment by Amirhakin** [comment]
Will the 9.0.3 update be released for PS4?

**Comment by Masamune3210** [comment]
What are you talking about?

**Comment by Dreamcast2** [comment]
Could you change Cheat Memu to the other position, it is easy to press the wrong button to close the application，many thanks!

**Comment by Zonemo001** [comment]
i try to activate cheats on any games i have my PS4 is crashing and shutting down. Fix problem plzzzz the ps4 is crashing

**Comment by Zonemo001** [comment]
When I want to use the cheat, the device turns off, please fix problme

**Comment by jonesmith518** [comment]
> 

have you tried to long press the share button?

**Comment by Zonemo001** [comment]
Yes, I tried it, but the device crashe

Le sam. 19 mars 2022 à 04:46, jonesmith518 ***@***.***> a
écrit :

**Comment by Zonemo001** [comment]
Yes, I tried it, but the device 
crashing

**Comment by jonesmith518** [comment]
got the klog  and hope it helps 
steps to reproduce the problem:
* goldhen 2.2 -> start game :  [CUSA18278] cyberpunk 2077 for example-> wait until game loads -> long press Share button -> panic or reboot 
* klog in the file below :
[error-log-panic-when-long-press-share-button-goldhen2.2.txt](https://github.com/GoldHEN/GoldHEN/files/8309496/error-log-panic-when-long-press-share-button-goldhen2.2.txt)

**Comment by Zonemo001** [comment]
Where do I find a file klog

**Comment by Metron7757** [comment]
Same here I'm having a very similar issue as a matter fact, I didn't start out with this issue. After updating to the latest version of Golden the cheat menu worked flawlessly! It was only until today it started happening with all of my games no matter what game I play. Ill load Of the game and as soon as I am playing it i'll go to activate cheats. It's not always a immediate crash. Sometimes the code will work for a few minutes and then the whole system will crash. It's very consistent though sometimes it'll crash instantly other times it'll just work for a A few minutes. The most consistent thing is that the same thing happens to all the games I play. This wasn't an issue in the beginning it just started happening. The only two things that I did differently this day was I just installed icon masks, and I followed a tutorial on how to make certain system apps not visible on my home screen. I believe The issue started when I mess with one of those two things. Because other than that up to this point it worked fine.

**Comment by Zonemo001** [comment]
Thanks a lot, I tried it, it works fine, thank you

**Comment by Zonemo001** [comment]
> The new version 2.2.1 should solve most of these problems, I close this jussue in case a new one will be created for the new version

Thanks a lot, I tried it, it works fine, thank you

**Comment by Zonemo001** [comment]
> The new version 2.2.1 should solve most of these problems, I close this jussue in case a new one will be created for the new version

The device has not yet turned off anyway

**Comment by Zonemo001** [comment]
ه‍

> The new version 2.2.1 should solve most of these problems, I close this jussue in case a new one will be created for the new version

Thanks for your interest. you are the best

**Comment by Zonemo001** [comment]
![Uploading IMG_20220322_214416.jpg…]()

**Comment by Zonemo001** [comment]
The device still turns off sometimes when the cheat is turned on

**Comment by Zonemo001** [comment]
I tried the new ،goldhen 2.2.1  but it turns off sometimes when I turn on the cheat. Plzzz fix

**Comment by Retroboy85** [comment]
I know this is old but not sure if it has been fixed because this is the first time I have ran into this problem.

I am on the latest version of GoldHen (stable not beta) and have GoldHen Cheats Manager installed and it is on the latest version with the latest updates for cheats and patches.

But when I try open the GoldHen Cheats Menu my console will completely freeze forcing me to power it off.

This will happen if I double press the PS button (which it has been set to) and if I hit PS button to go to the PS4 main screen and press Options then GoldHen Cheats Menu.

This is the only time I have noticed this and the game I am running is **_Cyberpunk 2027 (CUSA16596 v1.61)_**, wondering if it be a problem with one of the cheat files causing the problem, I connect to my GoldHen Cheats Directory via FTP and delete all cheat files for the game from its JSON/MC4/SHN directories so there are no cheats at all for the game.

But it still crashes when ever I try bring up the GoldHen Cheat Menu rather than showing the screen saying there are no cheats for this game.
